.github/workflows/development-workflow.yml

```diff
@@ -81,20 +81,20 @@ jobs:
               run: yarn workspace @app/server generate
```

Comment:
I think now is probably a good opportunity to move e2e tests into their own workflow? I'm not sure. I feel like that makes sense but I'm unsure of the tradeoffs. What is your opinion?

One workflow to test the build and another to test the e2e test.  Also, I think it makes sense to potentially scope the trigger based on paths. 

See https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onpushpull_requestpull_request_targetpathspaths-ignore

> Use the paths filter when you want to include file path patterns or when you want to both include and exclude file path patterns. Use the paths-ignore filter when you only want to exclude file path patterns. You cannot use both the paths and paths-ignore filters for the same event in a workflow.

Reply:
We could move it to another workflow but I don't think there would be any real benefit to doing it that way. And I did try to get some of the jobs to only run based on updates to a given path, but I had issues with it sometimes not working correctly (it would either not fire when that path was updated or it would fire when there wasn't an update to that path). I could have been doing something wrong but it seemed to work half the time so it's hard to know for sure.

Reply:
Okay, we'll keep that out of scope for this PR then. Lets just focus on separating out the workspace we can follow up with workflows later.